src/styles/blocks/header.scss

```
@@ -0,0 +1,67 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  align-items: center;
+  padding: 0 50px;
+
+  &__logo {
+    display: block;
+  }
+
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    &:not(:first-of-type, :last-of-type) {
+      margin: 0 10;
+    }
+
+    &:first-of-type {
+      margin-right: 10px;
+    }
+
+    &:last-of-type {
+      margin-left: 10px;
+    }
```

better to use one-direction margins
```suggestion
    margin-left: 20px;

    &:first-of-type {
      margin-left: 0;
    }
```